### PR TITLE
Update max number of nodes and pods for v1.6

### PR DESCRIPTION
**Pull request by boosty**
I could not find the current maximum for total containers and pods per node yet.

Since @johndmulhausen added the initial numbers via 97c728d343408f10c59dcfb7f2abf1d1f1051a1c, maybe he can help?

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3702)

### CODE REVIEW

**Comment by boosty** [comment]
I signed the CLA. Please check again.

**Comment by chenopis** [comment]
@thockin Are these updated numbers correct?

File: docs/admin/cluster-large.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ title: Building Large Clusters
...
+At {{page.version}}, Kubernetes supports clusters with up to 5000 nodes. More specifically, we support configurations that meet *all* of the following criteria:
 
-* No more than 2000 nodes
-* No more than 60000 total pods
-* No more than 120000 total containers
-* No more than 100 pods per node
+* No more than 5000 nodes
+* No more than 150000 total pods
+* No more than 120000 total containers (UPDATE?)
+* No more than 100 pods per node (UPDATE?)
```
Comment (gmarek) [inline docs/admin/cluster-large.md]: nope

File: docs/admin/cluster-large.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ title: Building Large Clusters
...
-At {{page.version}}, Kubernetes supports clusters with up to 1000 nodes. More specifically, we support configurations that meet *all* of the following criteria:
+At {{page.version}}, Kubernetes supports clusters with up to 5000 nodes. More specifically, we support configurations that meet *all* of the following criteria:
 
-* No more than 2000 nodes
-* No more than 60000 total pods
-* No more than 120000 total containers
-* No more than 100 pods per node
+* No more than 5000 nodes
+* No more than 150000 total pods
+* No more than 120000 total containers (UPDATE?)
```
Comment (gmarek) [inline docs/admin/cluster-large.md]: certainly 150k

**Comment by chenopis** [comment]
So, does total containers need to be updated?

**Comment by gmarek** [comment]
Yup, as I wrote there's no point of having number of Pods > number of Containers. We don't test big clusters with 2 containers/pod, so I'd just use the same number.

**Comment by boosty** [comment]
@gmarek Thanks for the feedback! I wonder why in the past the maximum number of containers was 2x the number of pods though.

@chenopis How should we proceed with this PR? In the master branch, the number of total containers got changed to 300000 via https://github.com/kubernetes/kubernetes.github.io/commit/a02d76a2c023ec9b60c4551c52c74bfe01fe83cc. Since you guys should be able to edit this PR (or just make the change in master and close this), please take over.

**Comment by gmarek** [comment]
OK, 300k SGTM then.

**Comment by chenopis** [comment]
@boosty Looks like these changes already made it into `master`, so after I resolved the conflict, the PR became a no-op.

**Comment by boosty** [comment]
@chenopis Thanks! But I am still wondering why the number was set to 300.000 in the end. If nobody tested it (as @gmarek said), wouldn't it be better to not have it in the docs at all?

**Comment by chenopis** [comment]
@boosty Let's ask @wojtek-t since, I think, he is in charge of this sort of testing and approved https://github.com/kubernetes/kubernetes.github.io/pull/4621.

**Pull request closed without merging**